### PR TITLE
Fix the Radio Sync Fix

### LINKED ISSUE #2939: [Bug]: Radio Station Syncing Problems
### Describe the bug

When I play with my friend without the menu, we have the same songs playing on the radio, but when I use YimMenu we have different songs playing at different times.

### Steps To Reproduce

1. Get into a vehicle with a player
2. Observe the song that is playing for you
3. Observe the song that is playing for them

### Expected Behavior

You are able to hear the same songs at the same time as other players.

### Logs and/or screenshots

N/A

### Store

Steam

### Language

English

### YimMenu Language

English (US)

### Additional context

May be OOM kick related, idk. I know that OOM kick protection blocks excessive radio sync requests.

### Build

- [X] I've confirmed the issue exists on the latest version of YimMenu

### Coload

- [X] I've tested this without co-loading anything.

### PR DESCRIPTION
**Pull request by Rxann**
closes #2939

thanks @lonelybud and @maybegreat48 for the code updates in the comments of the issue.

### CODE REVIEW

**Review by maybegreat48 — DISMISSED** [review]

**Comment by lonelybud** [comment]
can you move that `return true` to first if block level bcz `process` keeps returning true if things keep spamming whereas `exceeded_last_process` will only be called once in theory

**Comment by maybegreat48** [comment]
I didn't really notice. Thanks for catching that

**Comment by Rxann** [comment]
Github mobile is hard to use, i'll run the formatter when I get home and request a review then

File: src/hooks/protections/receive_net_message.cpp [hunk 1]
```diff
@@ -248,10 +248,10 @@ namespace big
 					{
 						// Make a translation for this new OOM kick protection
 						g_notification_service.push_error("PROTECTIONS"_T.data(), "OOM_KICK"_T.data());
-						return true;
 					}
+     return true;
```
Comment (Yimura) [inline src/hooks/protections/receive_net_message.cpp]: Was the formatter not used here or something? What is this indentation.

**Review by Yimura — DISMISSED** [review]

File: src/hooks/protections/receive_net_message.cpp (hunk 1 above)
Comment (Rxann) [reply src/hooks/protections/receive_net_message.cpp]: I tried to use github mobile, i'll run the formatted in a few minutes.

**Review by maybegreat48 — APPROVED** [review]
Looks OK now

**Review by Yimura — APPROVED** [review]

**Pull request merged**